### PR TITLE
Methods and results interpretations by @mbthese

### PR DESCRIPTION
**Pull request by mbthese**
I need help on explaining the equation.
I cannot seem to find the calculations, only the graphs.

### CODE REVIEW

File: 05-Variations.Rmd [hunk 1]
```diff
@@ -127,7 +127,16 @@ Subsequent analysis aimed to assess variations in leaf traits among species, ind
...
-*To describe.*
+Variance partitioning was obtained using linear mixed models for every trait and each pairs of levels studied. We aim to understand how much of the leaf trait variation is due to :   
+- measurement error, 
+- within-tree leaf variation, 
+- leaf sample repetitions among trees, 
+- intra- and inter-specific variation.
+
+*example for intra- and inter-specific variation*: We estimated the among-species variance in our trait of interest from the comparison of the observed variance within and between species. In this case, *Tree* is chosen to be a fixed effect and *Species* as random. 
+
+model <- lme(trait~1, random = ~1|Species, data)
```
Comment (sylvainschmitt) [inline 05-Variations.Rmd:138]: No, both are considered as random effect but tree is nested in species: https://github.com/sylvainschmitt/hydroITV/blob/71ee99c9f7ca53db0a3d51b6948e083b87e7ba21/05-Variations.Rmd#L102

Comment (sylvainschmitt) [reply 05-Variations.Rmd:138]: Update your pull request in consequence before acceptance.

**Comment by sylvainschmitt** [comment]
@mbthese , Great nice use of pull requests and code review ! 

Do you want me to include these first changes or should I wait for all the methods / interpretations?

**Comment by mbthese** [comment]
@sylvainschmitt thanks for the encouragements. Always happy to work with you!

You can include the first changes. I am going to read before further interpretation.

**Pull request merged**